src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/view/sw-first-run-wizard-finish/sw-first-run-wizard-finish.html.twig

```diff
@@ -1,58 +1,62 @@
 {% block sw_first_run_wizard_finish %}
 <div class="sw-first-run-wizard-finish">
+    <div v-if="!restarting">
+        <span class="sw-first-run-wizard-finish__bubble">
+            <mt-icon
+                class="sw-first-run-wizard-finish__icon"
+                name="regular-checkmark-xs"
+                size="18px"
```

Comment:
Same here, just curious if we can use tokens here too to properly scale the icons too